indico/modules/events/persons/templates/management/person_list.html

```diff
@@ -176,6 +176,15 @@
                             <i class="icon-stop colored-square no-account"></i>
                             <label for="filter-no-account">{% trans %}Users with no account{% endtrans %}</label>
                         </li>
+                        <li class="separator">
+                            <span class="titled-rule">{% trans %}or{% endtrans %}</span>
+                        </li>
+                        <li for="filter-no-registration">
+                            <input type="checkbox" id="filter-no-registration" data-filter="no_registration">
+                            <i class="icon-checkmark"></i>
+                            <i class="icon-stop colored-square no-registration"></i>
+                            <label for="filter-no-registration">{% trans %}Users who didn't register{% endtrans %}</label>
```

Comment:
Perhaps `Unregistered users` would better fit the dialog?

Reply:
I tried to stay consistent with "Users with no account", but "Unregistered users" sounds good as well